tests/resources/lsp_transpiler/config.yml

```diff
@@ -0,0 +1,11 @@
+remorph:
```

Comment:
TranspileConfig also has the same name, we need a different name for both or standardize both.

Reply:
Not sure this is correct, I can't find a `remorph` field in `TranspileConfig`...
See https://github.com/databrickslabs/remorph/blob/main/src/databricks/labs/remorph/config.py#L12C1-L25C1
Also no longer in `RemorphConfigs`...

Reply:
I meant the file name config.yml 

https://github.com/databrickslabs/remorph/blob/main/src/databricks/labs/remorph/config.py#L14

Reply:
ah. Fixed